src/main/view/StyleManager.cpp

```diff
@@ -7,10 +7,43 @@
 
 #include "StyleManager.h"
 
+#include <algorithm>
+
+#if defined(_WIN64) || defined(_WIN32)
+//Windows specific code
```

Comment:
Not sure we need this comment as the line above clearly reads that the code here will only be for Windows. 
Personally i'd only use comments if the code is not clear. Thoughts?